cmd/release-controller/sync_analysis.go

```diff
@@ -3,46 +3,58 @@ package main
 import (
 	"fmt"
 	imagev1 "github.com/openshift/api/image/v1"
-	"k8s.io/klog"
+	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	"time"
 )
 
-func (c *Controller) ensureAnalysisJobs(release *Release, releaseTag *imagev1.TagReference) error {
-	for name, analysisType := range release.Config.Analysis {
-		if analysisType.Disabled {
-			klog.V(2).Infof("Release %s analysis step %s is disabled, ignoring", releaseTag.Name, name)
-			continue
-		}
-		if analysisType.AnalysisJobCount == 0 {
-			klog.Warningf("Release %s analysis step %s configured without analysisJobCount, ignoring", releaseTag.Name, name)
-			continue
-		}
+const (
+	// defaultAggregateProwJobName the default ProwJob to call if no override is specified
+	defaultAggregateProwJobName = "release-openshift-release-analysis-aggregator"
+)
+
+func (c *Controller) verifyAnalysisConfig(verifyType ReleaseVerification) error {
+	if verifyType.MaxRetries > 0 {
+		return fmt.Errorf("verification analysis jobs cannot have retries")
+	}
+	return nil
+}
+
+func (c *Controller) launchAnalysisJobs(release *Release, verifyName string, verifyType ReleaseVerification, releaseTag *imagev1.TagReference, previousTag, previousReleasePullSpec string) error {
```

Comment:
verifyType is actually a releaseVerificationJobRunRequest ?

Reply:
verifyName is actually a prowJobName?

Reply:
`verifyName` is literally just the name of the validation test:
```
    "gcp":{
      "prowJob":{"name":"periodic-ci-openshift-release-master-ci-4.10-e2e-gcp"}
    },  
    "aws-serial":{
      "prowJob":{"name":"periodic-ci-openshift-release-master-ci-4.10-e2e-aws-serial"}
    },
```
So, `gcp` and `aws-serial` are values of `verifyName`